inc/ix/ethdev.h

```diff
@@ -222,7 +222,7 @@ struct ix_rte_eth_rss_conf {
 #define ETH_RSS_IPV6_UDP    0x0080 /**< IPv6/UDP packet. */
 #define ETH_RSS_IPV6_UDP_EX 0x0100 /**< IPv6/UDP with extension headers. */
 /* Definitions used for redirection table entry size */
-#define ETH_RSS_RETA_NUM_ENTRIES 128
+#define ETH_RSS_RETA_NUM_ENTRIES 512
 #define ETH_RSS_RETA_MAX_QUEUE   16
```

Comment:
As for the `ETH_RSS_RETA_MAX_QUEUE`, IX wants an RX queue per thread. I am not sure that we have a good reason for capping the number of queues to 16 except that our servers have 16 HTs. Don't worry about it for this pull request. We might want to make it more flexible (support servers with more HTs) later on, though.